index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <nav class ="main-nav">
+        <div class="logo">
+        <a href="#"><p>Worash Aboche</p></a>  
+        </div>
+         <a href="#"> <img src ="./images/Icon-Menu.svg" alt="menu" class="menu"> </a>
+        </nav>
```

Comment:
- [ ] Please use CSS to set a background for your design 

![image](https://user-images.githubusercontent.com/74744787/164340441-d170db41-4e79-4a18-8f3a-4f7658b20e1a.png)

- [ ] Please make sure to fix this weird distance between the background and the paragraph and make it looks exactly as in Figma design 

| Your design | Figma design |
| ------------- | --------------- |
| ![image](https://user-images.githubusercontent.com/74744787/164340606-c36bc3b8-c937-4715-945a-01cd3c3de73a.png) | ![image](https://user-images.githubusercontent.com/74744787/164340654-cf1d45dd-b144-46a0-b09d-d1842703079c.png) |